packages/react-storage/src/components/StorageBrowser/composables/Search.tsx

```diff
@@ -18,9 +18,15 @@ const TOGGLE_BLOCK = 'toggle';
 
 export interface SearchProps {
   onSearch: (term: string, includeSubfolders: boolean) => void;
+  searchPlaceholder?: string;
```

Comment:
Noted elsewhere, the `searchPlaceholder` changes should be removed

Reply:
We have different placeholders for LocationsView, LocationDetailView and CopyView. Is there a better way to handle that?

Reply:
That's the use case for the controls context, it allows for providing low level props to the composables without impacting their top level APIs

Reply:
I am providing the `searchPlaceholder` through ControlsContext in SearchControl [here](https://github.com/aws-amplify/amplify-ui/pull/6005/files#diff-c88f1b5bd17c3ba59ce3087ecf9abb609b23259d28b691374d65ebae22036a14R13), I don't think Search composable itself should use ControlContext ?

Reply:
We can follow up on this

Reply:
Thx, i'd like to understand this properly